cmd/spec-gen/main.go

```diff
@@ -23,7 +23,7 @@ import (
 	"strings"
 
 	"github.com/go-openapi/spec"
-	kfsvcv1alpha1 "github.com/kubeflow/kfserving/pkg/apis/serving/v1alpha2"
```

Comment:
Should we just import these as kfserving? Simplifies imports.

Reply:
Updated, thanks.